logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -666,6 +675,8 @@ abstract class UserSessionScopeCommon internal constructor(
             slowSyncManager
 
             callRepository.updateOpenCallsToClosedStatus()
+
+            observeNotificationTokenUpdating()
         }
```

Comment:
Maybe we could launch this on its own job, so people don't forget that this operation is long suspending.

```suggestion
        }
        launch {
            observeNotificationTokenUpdating()
        }
```